handler/src/test/java/io/netty/handler/timeout/WriteTimeoutHandlerTest.java

```diff
@@ -27,6 +26,7 @@
 
 public class WriteTimeoutHandlerTest {
 
+    // TODO: Remove ?
     @Test
     public void testPromiseUseDifferentExecutor() throws Exception {
```

Comment:
It's not really testing what it used to anymore. What's supposed to be tested is really that a promise being completed in one event loop and notify completions in another. I think that belongs with the `DefaultPromiseTest` and not so much here. The timeout handler is worth testing, though.

Reply:
Actually it was about that we use the correct eventloop for notification. Let me just remove this one for now as like you said it not really matters anymore